pkgs/development/misc/avr-gcc-with-avr-libc/default.nix

```diff
@@ -26,8 +26,8 @@ stdenv.mkDerivation {
   
   buildInputs = [ gmp mpfr libmpc zlib ];
   
-  # Make sure we don't strip the libraries in lib/gcc/avr.
-  stripDebugList= [ "bin" "avr/bin" "libexec" ];
+  # Make sure we don't strip the libraries in lib/gcc/avr and in avr/lib.
```

Comment:
@layus IIRS, I wrote that original dontStripList, not because it failed to strip, but because to broke the toolchain. Actually my experience is that "strip" doesn't particularly care about the CPU architecture, it can strip symbols from "other" architectures.